python_da/dq_checks/src/app.py

```diff
@@ -198,6 +222,43 @@ def upload_results_to_azure(result):
         logger.error(f"Unexpected error while uploading result to Azure Blob Storage: {str(e)}")
         raise e 
 
+def upload_image_to_azure(img_data, blob_name, container_name_images, connection_string):
```

Comment:
can you create a folder from code, I think if you give container path like ='/contaninername/somestatic value'

Reply:
Agreed, I'll create jiras I'll add it to the refactoring list. Tbh need to do a greater refactor as python files are getting a bit large. Thinking of adding a lint action to our code next week:
Will help ensure that our code is good quality. 
https://github.com/marketplace/actions/lint-action